### PR TITLE
Link to get_metrics.forecast_sample

### PR DESCRIPTION
**Pull request by seabbs**
+ spelling

### CODE REVIEW

File: sessions/forecast-evaluation.qmd [hunk 1]
```diff
@@ -152,7 +152,7 @@ sc_scores
 ::: {.callout-note collapse="true"}
 ## Learning more about the output of `score()`
 
-See the documentation for `?get_metrics.forecast_sample` for information on the default metrics for forecasts that are represented as samples (in our case the samples generated by the stan model).
+See the documentation for `get_metrics.forecast_sample()` for information on the default metrics for forecasts that are represented as samples (in our case the samples generated by the stan model).
```
Comment (sbfnk) [inline sessions/forecast-evaluation.qmd:155]: did you mean to insert an actual link, i.e.

```suggestion
See the documentation for `get_metrics.forecast_sample()`[https://epiforecasts.io/scoringutils/reference/get_metrics.forecast_sample.html] for information on the default metrics for forecasts that are represented as samples (in our case the samples generated by the stan model).
```

or else why this change?

Comment (seabbs) [reply sessions/forecast-evaluation.qmd:155]: So in my head this was `pkgdown` and so this would auto-link but now I realise it isn't and it won't

**Review by sbfnk — APPROVED** [review]
